### ISSUE TITLE
Etrange il manque https://www.education.gouv.fr/

### ISSUE THREAD

**Issue by jbledevehat** [bzg/gouvfrlist#5]
Dans ton fichier tested.gouv.fr.txt il manque l'URL https://www.education.gouv.fr/

**Comment by jbledevehat** [comment]
Il semble aussi manqué : 
http://www.enseignementsup-recherche.gouv.fr/

**Comment by bzg** [comment]
Effectivement. Il faut de toutes façons que je consolide la façon de collecter les adresses initiales.
